### PR TITLE
Beta 2023.05.26

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 13/13_01_01_CIEL13_5.txt [hunk 1]
```diff
@@ -784,9 +784,9 @@ Putrefying panoramas in patchwork mosaic.
 [sha:edb1f81fdd9883e3a823e9f96f06b2fc94f822f4]{
 -- Page 562, Offset 34889.
 -- 一夜ばかりの幻も　終わらぬものなら不治の<恋|どく>。
-%{i}Incurable %{/i}
-<%{i}poison%{/i}|%{i}l o v e   %{/i}>
-%{i} be, the never-ending night of fantasy.%{/i}
+%{i}
+Incurable poison be, the never-ending night of fantasy.
```
Comment (rschlaikjer) [inline script/Ciel/Day 13/13_01_01_CIEL13_5.txt:788]: Sucks to lose the ruby here, but it's either italics or the ruby. Can't have both.

Comment (yk1512) [reply script/Ciel/Day 13/13_01_01_CIEL13_5.txt:788]: I honestly would prefer adding a (love) in brackets right after, even if it is a little tacky. The omission feels like a big loss.

Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_5.txt:788]: Brackets would crush the mood here. I consider it unacceptable.
That said, it is 惜しい to have to lose it. Not sure what the technical problem here is exactly, but I suppose it'll have to be dealt with by coming up with a new rhyme. It's all I can think of. Obviously that's a harder ask. Perhaps using a different rhyme scheme is needed.

Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_5.txt:788]: archaism
>The poison of love hath no mending, 
wert the night of fantasy never-ending.

un-archaic
>The poison of love has no mending
>were the night of fantasy never-ending.

Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_01_CIEL13_5.txt:788]: Yeah tech problem is that the ruby text code is different to the normal path, so Ben would have to figure out how to patch that too.

Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_01_CIEL13_5.txt:788]: Forgot I had another trick up my sleeve. The ruby gets to stay.
2023-05-29-224642_2160x1286_scrot

**Pull request merged**